lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -269,7 +264,19 @@ static CanGenericSignature getConstrainedDerivativeGenericSignature(
           derivativeGenSig,
           /*addedGenericParams*/ {},
           std::move(requirements)},
-      nullptr)->getCanonicalSignature();
+      nullptr);
+}
+
+// Returns the canonical derivative generic signature for the given
```

Comment:
1. Three slashes for doc comment. 
2. “Returns” -> “Return” for consistency.

Reply:
> 1. Three slashes for doc comment.

Thanks for the catch, fixed.

> 2. “Returns” -> “Return” for consistency.

"Returns" is actually used more frequently than "Return" in this file (26 vs 4). I would prefer "Returns" in line with Apple Swift documentation ([example](https://developer.apple.com/documentation/swift/array/2994747-randomelement)). What would you prefer?

Reply:
Most functions in this file are not using a third person plural. However, I’m +1 on using third person plurals.